tensorrt_llm/bench/dataclasses/reporting.py

```
@@ -90,41 +90,59 @@ def generate_statistics_summary(self) -> None:
+        num_accepted_drafts_percentiles = PercentileStats.from_iterable(
+            num_accepted_drafts) if num_accepted_drafts else None
+        draft_acceptance_rate_percentiles = PercentileStats.from_iterable(
+            draft_acceptance_rate) if draft_acceptance_rate else None
+        acceptance_length_percentiles = PercentileStats.from_iterable(
+            acceptance_length) if acceptance_length else None
 
         stats = BenchmarkStatistics(
             num_requests=num_requests,
             total_latency_ns=end_time - start_time,
             total_output_tokens=sum(output_tokens),
             total_input_tokens=total_input_tokens,
```

Ruff (0.12.2)

Undefined name `draft_tokens`

(F821)